x-pack/legacy/plugins/lens/public/indexpattern_plugin/field_item.tsx

```diff
@@ -181,6 +180,18 @@ export function FieldItem(props: FieldItemProps) {
               <span className="lnsFieldItem__name" title={field.name}>
                 {wrappableHighlightableFieldName}
               </span>
+
+              <EuiIconTip
+                anchorClassName="lnsFieldItem__infoIcon"
+                content={i18n.translate('xpack.lens.indexPattern.fieldStatsButton', {
+                  defaultMessage:
+                    'Click for more information about {fieldName}. Or drag to drop field into visualization.',
```

Comment:
```suggestion
                    'Click to see contents of {fieldName}. Or drag field into visualization.',
```

Reply:
Yeah, I'm working with @gchaps on some of this copy.